go.mod

```diff
@@ -169,6 +169,8 @@ require (
 
 // Below are the long-lived replace of the Cosmos SDK
 replace (
+	cosmossdk.io/api => github.com/cosmos/cosmos-sdk/api v0.0.0-20230820170544-1bd37053e0c0
```

Comment:
sorry to be nitpicky, but can we bump it directly in the go.mod and not add replaces.
Otherwise, we need to manually remove this in subsequent PR.